tensorflow/python/distribute/distribute_lib.py

```diff
@@ -1842,7 +1850,7 @@ class StrategyV1(StrategyBase):
   TensorFlow's partitioned variables (where a single variable is split across
   multiple devices) at this time.
   """
-
+  @deprecated(None,'This method is not available in TF 2.x.')
```

Comment:
This is the TF1 Strategy base object. I don't think we would like to have deprecation warnings for TF1 users.

Reply:
Reverted the changes for TF1 Strategy based objects.